pubsub.md

```diff
@@ -59,7 +59,7 @@ Go to the [exchanges](http://localhost:15672/#/exchanges) page and select the `p
 Now for that last and most exciting part - let's publish a message to our `pubsub` exchange and see what happenes:
 
 ```
-$ rabbitmqadmin publish exchange=pubsub routing_key="" payload="I'm spam!"
+$ rabbitmqadmin publish exchange=pubsub routing_key=null payload="I'm spam!"
```

Comment:
I'm pretty sure this is not required. Let's go over it together. I cherry picked the typo fix, so closing this PR.